packages/renderer/src/index.js

```diff
@@ -31,7 +31,7 @@ const pdf = (initialValue, cache) => {
 
   if (initialValue) updateContainer(initialValue);
 
-  const render = async (compress = true) => {
+  const render = async (compress = false) => {
```

Comment:
Is it possible to pass this through as an option from `renderToBuffer` or similar methods?

Reply:
why would we want to do that if it slows render by 50%?

Reply:
not render -- slows the _entire request_ down by 50%